### PR TITLE
Add an OAuth token in the gcsweb-private deployment.

### PR DESCRIPTION
**Pull request by droslean**
/cc @openshift/openshift-team-developer-productivity-test-platform 

Signed-off-by: Nikolaos Moraitis <nmoraiti@redhat.com>

### CODE REVIEW

File: core-services/ci-secret-bootstrap/_config.yaml [hunk 1]
```diff
@@ -1413,3 +1413,11 @@
     - cluster: app.ci
       namespace: ci
       name: release-controller-signature-signer
+- from:
+    oauth:
+      bw_item: gcsweb-private-oauth
```
Comment (alvaroaleman) [inline core-services/ci-secret-bootstrap/_config.yaml:1418]: what oauth token is this? GCS? Github?

Comment (droslean) [reply core-services/ci-secret-bootstrap/_config.yaml:1418]: GCS oauth. To be more specific, this token was generated from a specific service account with limited permissions. (read-only GCS storage)

Comment (alvaroaleman) [reply core-services/ci-secret-bootstrap/_config.yaml:1418]: Can you then name everything (bitward entry, secret, mount) to reflect that this is a GCP SA and what name it has? The fact that its an oauth token tells me nothing about it.

Comment (droslean) [reply core-services/ci-secret-bootstrap/_config.yaml:1418]: Its not a service account. Its the OAuth only. But I will change the name to be more specific. Thanks for suggesting this.

**Pull request merged**